packages/snaps-execution-environments/src/webview/utils/MockProxyMessageStream.ts

```diff
@@ -0,0 +1,96 @@
+import { BasePostMessageStream } from '@metamask/post-message-stream';
+import {
+  isJsonRpcNotification,
+  isJsonRpcRequest,
+  isJsonRpcResponse,
+  isPlainObject,
+} from '@metamask/utils';
+
+const noOp = () => undefined;
+
+type MockProxyMessageStreamArgs = {
+  name: string;
+  target: string;
+  targetOrigin?: string;
+  targetWindow?: Window;
+  write?: () => void;
+};
+
+export class MockPostMessageStream extends BasePostMessageStream {
```

Comment:
This looks like duplicated code from https://github.com/MetaMask/snaps/blob/main/packages/snaps-utils/src/test-utils/stream.ts#L11-L74

Can we not just use the same implementations for these tests?

Reply:
Roger that.